docs/features/cameras.md

```diff
@@ -7,20 +7,21 @@ permalink: /features/cameras
 ---
 
 # Cameras
+
 {: .no_toc }
 
 ---
 
-Fluidd (and FluiddPI) has built in support for web cameras.
+Fluidd (and FluiddPI) have built-in support for web cameras.
```

Comment:
has/have
`have` is used with the pronouns I, you, we, and they, while `has` is used with he, she, and it.

since "Fluidd" is a pronoun, I think it needs "have".  correct?

Reply:
Not my native language but I would say HAS.

> Have is used with the pronouns I, you, we, and they. Has is used with he, she, and it.

Reply:
This is a tricky one. Fluidd is an "it", so for that you would use "has". But if you mention 2 things, Fluidd and FluiddPi in this case, the "it" becomes a "they" in which case you would use "have". **BUT**, since the FluiddPi is in parentheses, it isn't part of the "they" and you are back to a singular Fluidd "has".

If you want to fix it, I would say you either drop the FluiddPi completely and say "has", or drop the parentheses and say "have". No one will care either way lol.  But the way it was is correct and your proposal is not correct.